### ISSUE TITLE
desperately needs a README, some docs, and get added to the coop

### ISSUE THREAD

**Issue by ec1oud** [dleslie/nanovg-egg#3]
I got pointed to your egg on IRC, but I don't understand what you intended it to depend on, to create the window (glfw or something like that?), and to provide that render-thunks parameter (is that what it is?) which you set up at the end of the examples.

**Comment by dleslie** [comment]
Heh, I sort of forgot that I was working on this; I became seriously side-tracked and haven't really touched Scheme.

It uses Allegro for window management.

`render-thunks` was just my way of collecting a bunch of drawing routines and dispatching them.

**Comment by dleslie** [comment]
I'll see about finishing up the work here.

**Comment by ec1oud** [comment]
Yeah I figured out after a while that those two scm files go together, and got it working. 

I'll send you a pull request soon with one that uses the glfw3 egg instead.

One issue I haven't figured out yet is why the text isn't antialiased when I'm using GLFW, although the vector graphics do have AA, and when using Allegro the text does too.

**Comment by dleslie** [comment]
I've fixed the compile warnings and got it placed on the coup.
